src/Places/DataAccess.ts

```diff
@@ -134,15 +135,26 @@ export async function getPlaceMedia(id: string): Promise<Medium[]> {
 }
 
 async function getFromApi(toBeFetchedFromAPI: string[]): Promise<any> {
-	const apiResponse = await StApi.get('places?' + stringify({
-		ids: toBeFetchedFromAPI.join('|')
+	const CHUNK_SIZE: number = 50;
+	const idsChunks = splitArrayToChunks(toBeFetchedFromAPI, CHUNK_SIZE);
+	const apiResponsesForChunks: ApiResponse[] = await Promise.all(idsChunks.map((idsChunk: string[]) => {
```

Comment:
this is hard to read, I would split it into multiple calls.

```ts
const requests = idsChunks.map()
const responses = await Promise.all(requests)
```

Reply:
second, this is weird variable name, I would use `chunkedResponses`. (also for some other variables)